perfkitbenchmarker/linux_benchmarks/iperf_vpn_benchmark.py

```diff
@@ -0,0 +1,235 @@
+# Copyright 2014 PerfKitBenchmarker Authors. All rights reserved.
```

Comment:
Related to the request to break this into smaller PRs. This benchmark could be separated out as a follow-on.

Reply:
Sorry for the bloated PR. This started as a single benchmark and grew a bit big. 
To clarify - is the preference to split into smaller commits, smaller PRs, or both? 
Logical dependencies are: 
|custom CIDR| <-- |VPN svc| <-- |VPN providers| <-- |VPN benchmark|
If these work better as separate PRs I'll close this one and resubmit separately in the above order.

Reply:
Sorry I mean smaller PRs. 
This order makes a lot of sense

Reply:
Thanks! Closing this in favour of the smaller PRs then...